### ISSUE TITLE
Feed not updated according to schedule 

### ISSUE THREAD

**Issue by drallgood** [gdavis/Mimir-Feedback#195]
### Are you sure this is a new issue?

- [X] I have confirmed this is a new issue

### iOS Version

16 beta 5

### Device Model

iPhone 13 Pro Max

### App Version

1.0.3 (103)

### What happened?

I subscribe to an hourly news podcast 
The update frequency is set to 30min. However I don't get the new episode for hours, e.g at 8:35 pm it was still showing the 6pm one
A manual refresh fixes it

### Bug prevalence

Always 

### How does it happen?

> Subscribe to NPR News Now
> Set feed refresh to 30min
> Wait

**Comment by gdavis** [comment]
Thank you for reporting this. I have another ticket open to revamp the scheduling of feed updates which will address this at the same time.

**Comment by gdavis** [comment]
@drallgood sorry for the delay on getting to this ticket, but this week I was able to rewrite much of the subscription updater code and improve it. I did find some some cases where updates might not behave as expected, and should be more more reliable once the next update is released.

I will also be adding background updates in the next update, which should make the app stay much more up to date even when it is not being used.
